dockers/docker-orchagent-cavm/copp.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
copp configuration is mainly driven from the user need, for example which types of packet should be prioritized, what the policer rate is. There maybe some cases when different platform may need to have different policer rate upper bound. But given the policer we current set, it should not be an issue.

@stcheng, can you add the run swssconfig in the orchagent start.sh?